### PR TITLE
1.2 changes to install

### PR DESCRIPTION
**Pull request by JoanneOFlynn2018**

### CODE REVIEW

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc [hunk 1]
```diff
@@ -134,7 +131,7 @@ Before you deploy {ProjectShort} on {OpenShift}, you must enable the catalog sou
 . Install a CatalogSource that contains the Service Telemetry Operator and the Smart Gateway Operator:
 +
 
-ifeval::["{build}" == "upstream"]
+
```
Comment (csibbitt) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc:134]: I think this condition should stay. If I understand correctly, this CatalogSource is only required upstream, and no customer CatalogSource is required downstream (Deleting L156-L176 :+1: )

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc [hunk 2]
```diff
@@ -153,52 +150,24 @@ spec:
...
-      interval: 30m
-EOF
-----
-endif::[]
+
 
 . To validate the creation of your CatalogSource, use the `oc get catalogsources` command:
 +
-ifeval::["{build}" == "upstream"]
+
```
Comment (csibbitt) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc:157]: Same here.

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc [hunk 3]
```diff
@@ -153,52 +150,24 @@ spec:
...
-NAME                   DISPLAY                 TYPE   PUBLISHER   AGE
-redhat-operators-stf   Red Hat STF Operators   grpc   Red Hat     62m
-----
-endif::[]
+
 
 . To validate that the Operators are available from the catalog, use the `oc get packagemanifest` command:
 +
-ifeval::["{build}" == "upstream"]
+
```
Comment (csibbitt) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc:170]: And here

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc [hunk 4]
```diff
@@ -335,7 +295,7 @@ spec:
   channel: stable-1.2
   installPlanApproval: Automatic
   name: service-telemetry-operator
-  source: redhat-operators-stf
+  source: redhat-operators
```
Comment (csibbitt) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc:298]: :+1:

File: doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc [hunk 5]
```diff
@@ -60,38 +60,23 @@ If you do not expect to install {Project} again, delete the CatalogSource. When
 
 .Procedure
 
+ifeval::["{build}" == "upstream"]
```
Comment (csibbitt) [inline doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc:63]: :+1:

**Review by csibbitt — COMMENTED** [review]
Looks like we can leave in a few conditionals to keep the downstream docs even simpler. Deferring to @leifmadsen to be sure, though.

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc (hunk 1 above)
Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc:134]: Hi @csibbitt we moved the ifeval to the top of the module (line 123/118) 
I'll build the doc again to make sure that I have this right!

Comment (csibbitt) [reply doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc:134]: Oh, shoot, I see it now! Thanks.

**Comment by leifmadsen** [comment]
Hrmmm not sure why the CI is failing. I'll have to look at this tomorrow. The docs are definitely building without issue locally.

**Comment by leifmadsen** [comment]
CI fixed.

**Comment by leifmadsen** [comment]
There is a `//TODO` item I will hopefully address today.

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc [hunk 6]
```diff
@@ -119,13 +113,16 @@ spec:
...
 == Enabling InfraWatch Catalog Source
-endif::[]
-ifeval::["{build}" == "downstream"]
-== Enabling Red Hat STF Catalog Source
-endif::[]
+
+
+//ifeval::["{build}" == "downstream"]
+//== Enabling Red Hat STF Catalog Source
+//endif::[]
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc:125]:
```suggestion
```

**Review by leifmadsen — APPROVED** [review]

**Pull request merged**